docs/user-guide.md

```diff
@@ -237,11 +240,55 @@ INFO Successfully deleted deployment: frontend
 Note:
 - You must have a running Kubernetes cluster with a pre-configured kubectl context.
 
-## Alternate formats
+## Building and Pushing Docker Images
+
+Kompose supports both building and pushing Docker images. When using the `build` key within your Docker Compose file, your image will automatically be built and pushed.
+
```

Comment:
Maybe it's clear, but it might be good to add sentence explaining where the image will be pushed to.